tutorials/docker-gcplogs-driver/index.md

```diff
@@ -151,6 +151,70 @@ to use the Google Cloud logging driver when it is started by systemd:
     the [Logs Viewer](https://console.cloud.google.com/logs/viewer) as you did
     before.
 
+## Persisting configuration across reboots
+
+On Container-Optimized OS, files in `/etc/` are writable, but [data does not
```

Comment:
Oh, that's my bad, I didn't realize it was a product name. It still reads pretty awkwardly. I think making it a link would help.